src/GraphQL/Builders/ISchema.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace GraphQL.Builders
+{
+    public interface ISchema
+    {
+        void RegisterRepository<TRepository>(TRepository repository)
```

Comment:
Yep, agree. Thinking about this now, I'd rather extract the whole ISchema bit out of the basic library and into my own project. :+1: